distributed/worker.py

```diff
@@ -875,15 +874,14 @@ def __init__(
 
         setproctitle("dask-worker [not started]")
 
-        if dask.config.get("distributed.worker.profile.enabled"):
```

Comment:
Has this been removed on purpose?

Reply:
Might be a side effect of unresolved conflicts

Reply:
I had suspected this as well, but my local git shows this PR as being based on https://github.com/dask/distributed/commit/6d85a8536ee02fd2a9ae52f50de030d1cc7afe7e, which introduced that change. So this might be a rebase gone wrong (in particular after looking at the original changes from https://github.com/dask/distributed/commit/b940a7360a1a21217fc53d57f2b8c3e29a2ef323).